frappe_io/www/docs/user/en/understanding-doctypes.md

```diff
@@ -392,3 +393,23 @@ records.
 >>> settings.notification_frequency
 'Daily'
 ```
+
+## Actions and Links
+
+> Added in Version 12.1
+
+**Actions**
+
+A DocType may also have `Daily Action` that will result in a button creation on the DocType View. Supported actions are:
+
+1. **Server Action**: This will trigger a whitelisted server action.
+
+**Links**
+
+A standard navigation aid to the DocType view is the `Links` section in the dashboard. This helps the viewer identify at a glance the what are the document types connected to this DocType and helps can quickly create new related documents.
```

Comment:
This helps the viewer identify at a glance the what are the document types connected to this DocType and helps can quickly create new related documents.

=>
This helps the viewer identify at a glance which document types are connected to this DocType and can quickly create new related documents.